modules/images/webp-uploads/image-edit.php

```diff
@@ -387,9 +388,8 @@ function webp_uploads_get_next_full_size_key_from_backup( $attachment_id ) {
  */
 function webp_uploads_restore_image( $attachment_id, $data ) {
 	$backup_sources = get_post_meta( $attachment_id, '_wp_attachment_backup_sources', true );
-
 	if ( ! is_array( $backup_sources ) ) {
-		return $data;
+		$backup_sources = array();
```

Comment:
@mitogh Just double-checking, what was the particular reason you've added this change? I'm not saying it's wrong, but I would think this would also work without this change? Just wanna make sure I'm not missing something.

Reply:
Sure, the main reason was that in case the backup source was not set up this would initialize it to a new empty array instead, in order to set up the references to the full-size images. 

Is an edge case, but wanted to make sure if the edge case happens the stored value is consistent, instead of having no value even if we have backup values available.